api/Notification.json

```diff
@@ -73,6 +74,7 @@
           },
           "webview_ios": {
             "version_added": false,
+            "partial_implementation": true,
             "notes": "Notifications are supported in web apps saved to the home screen."
```

Comment:
There is something wrong here, as `version_added: false` and `partial_implementation` contradict each other.